orchestrator-salt/src/main/resources/salt/salt/metering/scripts/metering_package_manager.sh

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
this should be `bash`, especially you are using the `function` keyword which is non-standard: https://github.com/koalaman/shellcheck/wiki/SC2112

Reply:
will keep this as is, probably will update this in later/different change